spec/commands/decidim/half_signup/admin/update_auth_settings_spec.rb

```diff
@@ -16,6 +16,13 @@
     )
   end
   let(:command) { described_class.new(auth_settings, form) }
+  let(:sms_gateway_service) { "Decidim::Verifications::Sms::ExampleGateway" }
+
+  before do
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`